cartography/intel/aws/ec2/network_interfaces.py

```diff
@@ -22,191 +22,233 @@ def get_network_interface_data(boto3_session, region):
     """
-
+    logger.debug("-> Instance to subnet")
     neo4j_session.run(
         ingest_network_interface_instance_relations, aws_update_tag=aws_update_tag,
     )
 
+
+@timeit
+def load_network_interface_load_balancer_relations(neo4j_session, aws_update_tag):
+    """
+    Creates (:LoadBalancer)-[:PART_OF_SUBNET]->(:EC2Subnet) if
+    (:LoadBalancer)--(:NetworkInterface)--(:EC2Subnet).
+    """
     ingest_network_interface_loadbalancer_relations = """
     MATCH (i:LoadBalancer)-[:NETWORK_INTERFACE]-(interface:NetworkInterface)-[:PART_OF_SUBNET]-(s:EC2Subnet)
     MERGE (i)-[r:PART_OF_SUBNET]->(s)
     ON CREATE SET r.firstseen = timestamp()
     SET r.lastupdated = {aws_update_tag}
     """
-
+    logger.debug("-> ELB to subnet")
     neo4j_session.run(
         ingest_network_interface_loadbalancer_relations, aws_update_tag=aws_update_tag,
     )
 
+
+@timeit
+def load_network_interface_load_balancer_v2_relations(neo4j_session, aws_update_tag):
+    """
+    Creates (:LoadBalancerV2)-[:PART_OF_SUBNET]->(:EC2Subnet) if
+    (:LoadBalancerV2)--(:NetworkInterface)--(:EC2Subnet).
+    """
     ingest_network_interface_loadbalancerv2_relations = """
     MATCH (i:LoadBalancerV2)-[:NETWORK_INTERFACE]-(interface:NetworkInterface)-[:PART_OF_SUBNET]-(s:EC2Subnet)
     MERGE (i)-[r:PART_OF_SUBNET]->(s)
     ON CREATE SET r.firstseen = timestamp()
     SET r.lastupdated = {aws_update_tag}
     """
-
+    logger.debug("-> ELBv2 to subnet")
     neo4j_session.run(
         ingest_network_interface_loadbalancerv2_relations, aws_update_tag=aws_update_tag,
     )
 
 
+@timeit
+def load(neo4j_session, data, region, aws_account_id, aws_update_tag):
+    elb_associations = []
+    elb_associations_v2 = []
+    instance_associations = []
+
+    for network_interface in data:
+        # https://aws.amazon.com/premiumsupport/knowledge-center/elb-find-load-balancer-IP/
+        matchObj = re.match(r'^ELB (?:net|app)/([^\/]+)\/(.*)', network_interface.get('Description', ''))
+        if matchObj:
+            elb_associations_v2.append({
+                'netinf_id': network_interface['NetworkInterfaceId'],
+                'elb_id': '{}-{}.elb.{}.amazonaws.com'.format(matchObj[1], matchObj[2], region),
+            })
+        else:
+            matchObj = re.match(r'^ELB (.*)', network_interface.get('Description', ''))
+            if matchObj:
+                elb_associations.append({
+                    'netinf_id': network_interface['NetworkInterfaceId'],
+                    'elb_name': matchObj[1],
+                })
+
+        if 'Attachment' in network_interface and 'InstanceId' in network_interface['Attachment']:
+            instance_associations.append({
+                'netinf_id': network_interface['NetworkInterfaceId'],
+                'instance_id': network_interface['Attachment']['InstanceId'],
+            })
+    load_network_interfaces(neo4j_session, data, region, aws_account_id, aws_update_tag)
```

Comment:
Previously, all Neo4j write actions lived in the same function. This PR separates them out to several functions as seen below so that execution is easier to trace.